translations/monero-core_nl.ts

```diff
@@ -1897,22 +2073,22 @@ Voor een betalingsbewijs hoeft u het adres van de ontvanger niet op te geven.</t
         <translation>Automatisch een node starten op de achtergrond (aanbevolen)</translation>
     </message>
     <message>
-        <location filename="../wizard/WizardDaemonSettings.qml" line="144"/>
+        <location filename="../wizard/WizardDaemonSettings.qml" line="148"/>
         <source>Blockchain location</source>
         <translation>Locatie van blockchain</translation>
     </message>
     <message>
-        <location filename="../wizard/WizardDaemonSettings.qml" line="151"/>
+        <location filename="../wizard/WizardDaemonSettings.qml" line="159"/>
         <source>(optional)</source>
         <translation>(optioneel)</translation>
     </message>
     <message>
-        <location filename="../wizard/WizardDaemonSettings.qml" line="170"/>
-        <source>Connect to a remote node until my own node has finished syncing</source>
-        <translation>Verbinding maken met een externe node totdat mijn eigen node is gesynchroniseerd</translation>
+        <location filename="../wizard/WizardDaemonSettings.qml" line="183"/>
+        <source>Bootstrap node (leave blank if not wanted)</source>
+        <translation>Externe node (leeg laten als u die niet gebruikt)</translation>
```

Comment:
Wellicht handig om deze wel `Bootstrap` te laten, gezien dit in de GUI een andere functionaliteit heeft dan puur het gebruik van een externe node.

Reply:
You'd look 'bootstrap' up, at which point you'll more easily find documentation regarding bootstrap nodes, as opposed to translating 'externe node', which yields totally different documentation. I'd also argue that there are many english words in the dutch language.

Your choice though.

Reply:
Good point! I will change it. I left *key image* untranslated for the same reason.